### DISCUSSION TITLE
duration of a traffic light

### DISCUSSION THREAD

**Discussion by Didoul35** [slyh/Cities2-TrafficLightsEnhancement#100] [Ideas]
Good morning,

Have you planned, or even started, the traffic light time management functionality?

If so, it could be interesting to keep your current systems but just to have a gauge (as you did for pedestrians) to facilitate the management of traffic light intersections, then then create a traffic light time management system. tricolors with personality settings by the user (like TMPE in CS1).

THANKS!

**Comment by slyh** [comment]
Yes, it's planned. I'm not sure if it can measure traffic flow, but at least it is easy to modify the minimal duration of a phase.

**Reply by Didoul35** [reply]
Hello, no news ? Have you succed to make modify the minimal duration or maximal of a phase ?
